### PR TITLE
docs: update button

### PR DESCRIPTION
**Pull request by StephanieTW**
## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/docs/liveEditorCode/alert.code.js [hunk 1]
```diff
@@ -3,13 +3,11 @@
   return dismissed ? null : (
     <Alert
       dismissible
-      size={Alert.Size.SMALL}
-      arrow={Alert.ArrowPosition.TOP_LEFT}
-      type={Alert.Type.WARNING}
+      size={Alert.Size.LARGE}
```
Comment (andreapiras) [inline packages/docs/liveEditorCode/alert.code.js:6]: Test: maybe we want to encourage small version

Comment (andreapiras) [inline packages/docs/liveEditorCode/alert.code.js:6]: MAybe we don't want to encourage LARGE
```suggestion
      size={Alert.Size.SMALL}
```

File: packages/docs/liveEditorCode/alert.code.js [hunk 2]
```diff
@@ -4,12 +4,10 @@
     <Alert
       dismissible
       size={Alert.Size.SMALL}
-      arrow={Alert.ArrowPosition.TOP_LEFT}
```
Comment (helendurranttw) [inline packages/docs/liveEditorCode/alert.code.js:7]: I've updated these examples in the Alert PR, so don't worry about changing here.

**Comment by andreapiras** [comment]
Close Test

**Pull request closed without merging**